doc/api/http.md

```diff
@@ -695,6 +695,10 @@ Once a socket is assigned to this request and is connected
 ### request.setTimeout(timeout[, callback])
 <!-- YAML
 added: v0.5.9
+changes:
+  - version: v9.0.0
+    pr-url: https://github.com/nodejs/node/pull/8895
+    description: set socket timeout when sockets connects
```

Comment:
It was like this even before https://github.com/nodejs/node/pull/8895. https://github.com/nodejs/node/pull/8895 only fixed a particular case inconsistent with the others.

Reply:
Yep, but this was the PR I found which changed the behaviour, so it seems reasonable to use that is the change description :)

I'm happy to link to something else if you like though.

Reply:
I'm fine with the link, it's the description that is a bit misleading imho as per https://github.com/nodejs/node/pull/25123#discussion_r242804236